files/en-us/mozilla/projects/nss/ssl_functions/old_ssl_reference/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 title: OLD SSL Reference
-slug: Mozilla/Projects/NSS/SSL_functions/OLD_SSL_Reference
+slug: Mozilla/Projects/en-US/docs/Mozilla/Projects/NSS/SSL_functions/OLD_SSL_Reference
```

Comment:
I wonder how this could have happened. Was it done manually? Or semi-manually with some massive search-and-replace in someone's IDE?
The `broken_links` flaw fixer should be smart enough to only do replacements when there's a `href="` prefix. 

Anyway, change this back and I think the tests should work. 

```suggestion
slug: Mozilla/Projects/NSS/SSL_functions/OLD_SSL_Reference
```

Reply:
Thank you. I must have done it in search and replace. Couldn't "see" it though. Sorry!